src/bin_scripts/ic-pack.ts

```
@@ -1,381 +1,384 @@
+              ? `@${packageScope}/${name}_server`
+              : `@${packageScope}/${name}_${canisterEnv}_server`;
+            const npm_server_input: PackNpmServerInput = {
+              did_file_path: did_path,
+              wasm_file_path: wasm_path,
+              target_dir_path: `${npm_dir}/server/${name}`,
+              name: packageName,
+              version: buildContext.icPackInput.version
+            };
+            pack_npm_server(npm_server_input);
+          }
         }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)